### ISSUE TITLE
Add more spelling termes used in Electrical domain

### ISSUE THREAD

**Issue by christiankral** [modelica/ModelicaStandardLibrary#2694]
In the Modelica documentation of `Modelica.UsersGuide.Conventions.Terms.Electrical` the spelling of terms are collected. This is a cumulative ticket to collect additional terms to be implemented in the future.

**Comment by christiankral** [comment]
Add

    <tr>
      <td><a href="http://www.electropedia.org/iev/iev.nsf/display?openform&amp;ievref=351-47-17">set-point</a></td>
      <td>set point, setpoint</td>
    </tr>

**Comment by christiankral** [comment]
<tr>
  <td><a href="http://www.electropedia.org/iev/iev.nsf/display?openform&amp;ievref=151-15-21">no-load</a></td>
  <td>noload, no load</td>
</tr>

**Comment by christiankral** [comment]
```
<tr>
  <td><a href="http://www.electropedia.org/iev/iev.nsf/display?openform&amp;ievref=151-11-10">electromechanical</a></td>
  <td>electro mechanical, electro-mechanical</td>
</tr>
```

**Comment by beutlich** [comment]
@christiankral Do you want to keep this issue open after #2854 got merged?

**Comment by christiankral** [comment]
Yes, that's a good idea. Let's keep it open, as there will come more terms along...

**Comment by christiankral** [comment]
How do you spell?

sub-package | subpackage | sub package

In the MSL I found sub-package and subpackage.

**Comment by beutlich** [comment]
[subpackage](https://en.wiktionary.org/wiki/subpackage)
